src/main/java/com/internet/shop/controllers/admin/GetUserOrdersController.java

```diff
@@ -12,8 +12,8 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-@WebServlet("/users/orders")
-public class GetCurrentUserOrdersController extends HttpServlet {
+@WebServlet("/admin/users/orders")
+public class GetUserOrdersController extends HttpServlet {
```

Comment:
```suggestion
public class GetSelectedUserOrdersController extends HttpServlet {
```

Reply:
But I think only administator should see any user orders so I put it to admin controllers